phpseclib/Common/Functions/Objects.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Common Object Functions
+ *
+ * PHP version 5
+ *
+ * @category  Common
+ * @package   Functions\Objects
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2016 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link      http://phpseclib.sourceforge.net
+ */
+
+namespace phpseclib\Common\Functions;
+
+/**
+ * Common Object Functions
+ *
+ * @package Functions\Objects
+ * @author  Jim Wigginton <terrafrost@php.net>
+ */
+abstract class Objects
```

Comment:
This of course a result of not properly designing an API and should not be fixed via hacking around using Reflection.